lib/ret_web/controllers/api-internal/v1/AuthTokenController.ex

```diff
@@ -0,0 +1,18 @@
+defmodule RetWeb.ApiInternal.V1.AuthTokenController do
+  require Logger
+
+  use RetWeb, :controller
+
+  def post(conn, %{"email" => email}) when is_binary(email) do
+    {:ok, token, _params} =
+      if !System.get_env("TURKEY_MODE") do
```

Comment:
It’s best-practice to quarantine all `System.get_env/1` calls.
```suggestion
      if not Ret.turkey_mode?() do
```

**lib/ret/ret.ex**
```elixir
@spec turkey_mode? :: boolean
def turkey_mode?,
  do: Application.fetch_env!(:ret, :turkey_mode?)
```

**config/config.exs**
```elixir
config :ret,
  ecto_repos: [Ret.Repo, Ret.SessionLockRepo],
  turkey_mode?: System.get_env("TURKEY_MODE") === "1"
```

Reply:
thx for the code lol
commited

Reply:
No problem! You’ll want to make sure to update `ret.ex` and `config.exs` alongside `AuthTokenController.ex`.